src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -72,6 +72,32 @@
  * <pre>
  * &lt;module name=&quot;MissingOverride&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test extends SuperClass {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     &#64;Override
+ *     public void test1() { // OK
+ *
+ *     }
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void test2() { // violation
+ *
+ *     }
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     private void test3() { // violation
+ *
+ *     }
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public static void test4() { // violation
```

Comment:
Violation for `test2` is "no override", but for private and static methods violation will be different. Please add violation comments to show it for all examples above.

Reply:
I am done.